integrations/README.md

```diff
@@ -9,3 +9,5 @@ An **Integration** is a connection to an external system that stores data for yo
 Aqueduct comes with a pre-built set of integrations to common data systems to reduce the manual engineering work needed to deploy a prediction pipeline. We have support for most common relational databases and for AWS S3.
 
 We're constantly working to add support for new integrations, so if there's something you'd like to see, let us know on our community Slack or on [GitHub](https://github.com/aqueducthq/aqueduct/issues/new).
+
+You can play around with our [demo integration](./aqueduct-demo-integration.md). Also check our [quick start guide](./quick-start-guide.md) to add your own integration and get familiar with our integration management UI.
```

Comment:
"Also check our [quick start guide](./quick-start-guide.md) to add your own integration and get familiar with our integration management UI." --> "Also check our [quick start guide](./quick-start-guide.md) to get familiar with our integration management UI." Adding an integration is part of getting familiar so I don't think we need to explicitly mention it.

Reply:
Also, to be more clear, we should explicitly call the quick start guide the "Integration Management Quick Start Guide" since it only covers integration management and hopefully we will have more guides in the future.